src/styles/index.scss

```
@@ -1,3 +1,74 @@
-body {
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+@mixin on-hover {
+  background-color: #ffbfcb;
+  cursor: pointer;
+  transform: translateY(-20px);
+  transition-duration: 500ms;
+}
+
+$cell-size: 100px;
+$cell-color: #eee;
+
+.container {
+  display: flex;
+  width: 100vw;
+  height: 100vh;
+  align-items: center;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  max-width: 726px;
+  padding: 10px;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+    height: $cell-size;
+    width: $cell-size;
+    border: 1px solid black;
+    background-color: $cell-color;
+
+    &::before {
+      position: absolute;
+      content: '';
+      font-size: 30px;
+    }
+
+    &:hover {
+      @include on-hover;
+    }
+  }
+}
+
+@for $i from 1 through 31 {
+  .calendar__day:nth-child(#{$i})::before {
+    content: '#{$i}';
+  }
+}
+
+@for $i from 28 through 31 {
+  .calendar--month-length-#{$i} .calendar__day:nth-child(n + #{$i + 1}) {
+    display: none;
+  }
+}
+
+@each $var1, $var2 in (mon: 1, tue: 2, wed: 3, thu: 4, fri: 5, sat: 6, sun: 7) {
```

Move it to variable `(mon: 1, tue: 2, wed: 3, thu: 4, fri: 5, sat: 6, sun: 7)` and use it here
```suggestion
@each $var1, $var2 in $days-week {
```